HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -127,33 +127,55 @@ private void saveMap()
 
         private void Edit_Display_Information(object sender, RoutedEventArgs e)
         {
+            this.chooser.SelectedItem = DisplayMapObject.MapObjectType.ObjectTypeFullName;
             EditMode = !EditMode;
         }
 
         public void ChangeEditButtonVisibility() {
             if (_selectedMapObject == null)
             {
                 EditObjectButton.Visibility = Visibility.Hidden;
+                Save.Visibility = Visibility.Hidden;
+                EditMode = false ;
             }
             else
             {
                 if (_selectedMapObject.MapObjectEntity.MapObjectType.TypeOfMapObject != TypeOfMapObject.ROAD)
                 {
                     EditObjectButton.Visibility = Visibility.Visible;
+                    Save.Visibility = Visibility.Visible;
                 }
                 else {
                     EditObjectButton.Visibility = Visibility.Hidden;
+                    Save.Visibility = Visibility.Hidden;
```

Comment:
@kmilena98 I would definitely rename "Save" to "SaveButton". It is more precise, concise and readable.